README.md

```diff
@@ -28,6 +28,7 @@ Check here for a current list of [**known issues**](#known-issues) for Azure Pol
 - [How to assign policies using the Azure portal](https://docs.microsoft.com/azure/governance/policy/assign-policy-portal)
 - [How to assign policies using Azure PowerShell](https://docs.microsoft.com/azure/governance/policy/assign-policy-powershell)
 - [How to assign policies using Azure CLI](https://docs.microsoft.com/azure/governance/policy/assign-policy-azurecli)
+- [Export and manage Azure policies as code with GitHub](https://docs.microsoft.com/en-us/azure/governance/policy/tutorials/policy-as-code-github)
```

Comment:
Please remove 'en-us' from the url

Reply:
Done. Thanks for catching this 👍

Reply:
Updated to "Azure Policy resources". Thanks! - @DCtheGeek